extensions/amp-analytics/0.1/test/test-transport.js

```diff
@@ -101,6 +103,44 @@ describes.realWin(
       expectImagePixel('https://example.test/test', 'no-referrer');
     });
 
+    it('carries attributionSrc over to image pixel', () => {
+      setupStubs(true, true);
+      env.sandbox
+        .stub(privacySandboxUtils, 'isAttributionReportingAllowed')
+        .returns(true);
+      sendRequest(win, 'https://example.test/test', {
+        beacon: true,
+        xhrpost: true,
+        image: true,
+        referrerPolicy: 'no-referrer',
+        attributionsrc: 'https://example.test/attributionsrc',
```

Comment:
are there any documentation that needs to be updated?

Reply:
We'll likely have to update https://github.com/ampproject/amphtml/blob/main/extensions/amp-analytics/amp-analytics.md#transport. Is this something that I need to do or is there someone on the amp team to update documentation that I need to talk to?